### DISCUSSION TITLE
could not delect file format when import EBSD data

### DISCUSSION THREAD

**Discussion by Y-i-L-i** [mtex-toolbox/mtex#2269] [Ask Anything]
import a EBSD data from Bruker acquisition software will error as follow
<img width="289" alt="b09ebb144b124f88ff6b4fb48271e72" src="https://github.com/user-attachments/assets/38f10a29-e47d-468f-bedc-a3e62b68f09d">
How should I solve this problem?

**Comment by kilir** [comment]
Hi,

could you provide a small example file?

Cheers,

Rüdiger
